setup.py

```diff
@@ -18,7 +18,7 @@
     packages=setuptools.find_packages(),
     package_data={},
     install_requires=[
-        'numpy',
+        'numpy','bs4'
     ],
```

Comment:
Hi.
The idea in spikeextractors is that every new extractor have only weak dependency.
In short, we have in setup.py very few dep.

The method we use is something like that

```python
try:
    import bs4
    HAVE_BS4 = True
except ImportError:
    HAVE_BS4 = False


class NeuroscopeRecordingExtractor(RecordingExtractor):
    def __init__(self):
        assert HAVE_BS4, 'please install bs4 to use this extractor'
```



```

Reply:
Fixed!